src/NuGet.Services.SearchService/Settings/local.json

```diff
@@ -11,7 +11,8 @@
     "AuxiliaryDataStorageDownloadsPath": "downloads.v1.json",
     "AuxiliaryDataStorageVerifiedPackagesPath": "verifiedPackages.json",
     "AuxiliaryDataReloadFrequency": "01:00:00",
-    "AuxiliaryDataReloadFailureRetryFrequency": "00:00:30"
+    "AuxiliaryDataReloadFailureRetryFrequency": "00:00:30",
+    "DeploymentLabel": "1.0.0-zlocal"
```

Comment:
As far as I know deployment labels are generated by EV2. Is the intention here for EV2 to place the deployment label in the config file?

Reply:
This config file is only observed locally (thus the `local.json` file name). It is used for debugging on your dev box. In the startup of the app, we define that environment variables (i.e. app settings in Azure Web Apps) take precedence over this file. Ev2 will place all app settings, including its deployment label as environment variables, so this `1.0.0-zlocal` value will never be seen in production.